alfresco-transform-misc/alfresco-transform-misc-boot/src/test/java/org/alfresco/transformer/MiscControllerTest.java

```diff
@@ -245,19 +245,21 @@ public void testRFC822NestedAlternativeToText() throws Exception
     public void testExtractMetadataRFC822() throws Exception
     {
         String expected =
-                "{\"{http://www.alfresco.org/model/content/1.0}addressee\":\"Nevin Nollop <nevin.nollop@gmail.com>\"," +

```

Comment:
Formatting / line-endings change ? 

As a workaround, I've switched to:

https://github.com/Alfresco/alfresco-transform-core/pull/372/files?w=1

Reply:
Yes, switching to a `TreeMap` in the `AbstractMetadataExtractor` meant that the expected response for these tests needed to be re-ordered into descending order of Key.